sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StreamingInserts.java

```diff
@@ -38,20 +38,39 @@
   private InsertRetryPolicy retryPolicy;
 
   /** Constructor. */
-  StreamingInserts(CreateDisposition createDisposition,
+  public StreamingInserts(CreateDisposition createDisposition,
                    DynamicDestinations<?, DestinationT> dynamicDestinations) {
+    this(createDisposition, dynamicDestinations, InsertRetryPolicy.alwaysRetry());
+  }
+
+  /** Constructor. */
+  public StreamingInserts(CreateDisposition createDisposition,
```

Comment:
Done. The second constructor must be private, as our API surface tests aren't happy if BigQueryServices is exposed in a public method.